File: validation/differential/corpus/layer1/go/service.go
// Package demo does things.
package demo

import "strings"

// Service handles requests.
type Service struct {
	Marker string
}

func (s *Service) Handle(request string) string {
	raw := `// not a comment
still raw`
	return strings.Join([]string{s.Marker, request, raw}, "")
}

func Helper() int { return 1 }

File: validation/differential/corpus/layer1/java/Service.java
// Header comment.
package demo;

/* A block comment
   spanning two lines. */
public class Service {
    private final String marker = "// not a comment";

    public Service() { }

    public String handle(String request) {
        return marker + request; // trailing
    }

    interface Callback { void done(); }

    enum Mode { FAST, SLOW }

    record Pair(int left, int right) { }
}
